packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -340,6 +342,10 @@ class _DatePickerDialogState extends State<_DatePickerDialog> {
     setState(() => _selectedDate = date);
   }
 
+  void _handleResetDate() {
+    setState(() => _selectedDate = _resetDate);
+  } 
```

Comment:
```suggestion
  }
```

Reply:
Remove trailing spaces to get analyzer to pass the checks.

Reply:
Thank you

Reply:
Why anyone can not review for my PR??

Reply:
Flutter team might be busy with some higher priority work, Your PR review process is queued as **HansMuller** requested a review from **darrenaustin** . 

Meanwhile you can work on tests, to avoid some unnecessary edits by someone else afterwards.

Also by merging latest master, other CI checks may pass as some of the check failures are unrelated to your PR. Thanks!